torch/testing/_internal/common_methods_invocations.py

```diff
@@ -15888,6 +15910,14 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
         sample_inputs_func=sample_inputs_grid_sample,
         supports_gradgrad=False,
         gradcheck_nondet_tol=1e-15),
+    OpInfo(
+        "grid_sampler_2d",
```

Comment:
Is there a reason this isn't already covered by the `grid_sample` OpInfo?

Reply:
Adding this OpInfo because grid_sample aot_dispatch test stills fails but due to (grid_sampler3d). We can remove this once we add meta support for that.